page-objects/src/components/sidebar/ViewTitlePart.ts

```diff
@@ -38,25 +38,23 @@ export class ViewTitlePart extends ElementWithContexMenu {
      * @returns Promise resolving to TitleActionButton object
      */
     async getAction(title: string): Promise<TitleActionButton> {
-        return new TitleActionButton(title, this);
+        return new TitleActionButton(TitleActionButton.locators.ViewTitlePart.actionContstructor(title), this);
     }
  }
 
  /**
   * Page object representing a button inside the view title part
   */
  export class TitleActionButton extends AbstractElement {
-    private title: string;
 
-    constructor(title: string, viewTitle: ViewTitlePart) {
-        super(TitleActionButton.locators.ViewTitlePart.actionContstructor(title), viewTitle);
-        this.title = title;
+    constructor(actionConstructor: By, viewTitle: ViewTitlePart) {
+        super(actionConstructor, viewTitle);
     }
 
     /**
      * Get title of the button
      */
-    getTitle(): string {
-        return this.title;
+    async getTitle(): Promise<string> {
```

Comment:
Potential major change in semver version.

Reply:
Yes, we must count with that. Thanks for noticing.